packages/shared/src/hooks/useBoot.ts

```diff
@@ -0,0 +1,20 @@
+import { useQueryClient } from 'react-query';
+import { BOOT_QUERY_KEY } from '../contexts/common';
+import { squadMemberships } from '../graphql/squads';
+import { Boot } from '../lib/boot';
+
+type UseBoot = {
+  updateSquads: () => Promise<void>;
+};
+
+export const useBoot = (): UseBoot => {
+  const client = useQueryClient();
+  const updateSquads = async () => {
+    const squads = await squadMemberships();
```

Comment:
Alternatively as we know which one gets deleted we can manually delete is from the cache.
Thinks upvotes and bookmark use this concept, takes away a API call where we know the result.

deleteCommentCache is also a neat function that uses this approach a bit more.

Reply:
No worries, it makes sense. Done 👍